docs/common/ca.md

```diff
@@ -125,18 +103,17 @@ In addition to the above CAs, you can install other CAs via RPM. These only work
 Managing Certificate Revocation Lists
 -------------------------------------
 
-In addition to CA certificates, you must have updated Certificate Revocation Lists (CRLs) which are are lists of certificates that have been revoked for any reason. Software in the OSG Software Stack uses CRLs to ensure that you are talking to valid clients or servers.
+In addition to CA certificates, you must have updated Certificate Revocation Lists (CRLs). CRLs contain a certificate blacklists that OSG software uses to ensure that your hosts are talking to valid clients or servers. To maintain up to date CAs, you will need to run the `fetch-crl` services. 
```

Comment:
"contain a certificate blacklists" -> "contain certificate blacklists"

I feel like "hosts are talking to valid clients or servers" would be more accurate as either "hosts are not talking to invalid clients or servers" or "hosts are only talking to valid clients or servers", since that's more like how a blacklist works.